spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/jooq/JooqTestIntegrationTests.java

```diff
@@ -43,7 +41,6 @@
  */
 @RunWith(SpringRunner.class)
 @JooqTest
-@TestPropertySource(properties = "spring.jooq.sql-dialect = H2")
```

Comment:
Ok, I get that now (the change of perspective), I would expect the same. Thanks.